test/integration/test.visual_recognition.prepop.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+const fs = require('fs');
+const nock = require('nock');
+const watson = require('../../index');
+const assert = require('assert');
+const path = require('path');
+const authHelper = require('./auth_helper.js');
+const auth = authHelper.auth;
+const describe = authHelper.describe; // this runs describe.skip if there is no auth.js file :)
+
+const THIRTY_SECONDS = 30000;
+const TWO_SECONDS = 2000;
+
+const logit = function(string) {
+  // console.log('==> ' + string);
+  return string;
+};
+
+describe('visual_recognition_integration_prepopulated', function() {
+  // ugh.
+  this.timeout(THIRTY_SECONDS * 8);
+  this.slow(TWO_SECONDS);
+  this.retries(5);
+
+  let visual_recognition;
+  let classifier_id;
+
+  const test_training_status = function(resolve, reject) {
+    //  This evil recursive function will be used to verify that the classifier
+    //  has finished training. 'resolve' and 'reject' are functions from an
+    //  enclosing promise (or a follow-on callback for resolve if you prefer)
```

Comment:
I may end up moving this to the lib itself, because I can imagine end-users finding it useful also.